cmd/lint/vault.go

```diff
@@ -0,0 +1,74 @@
+package lint
```

Comment:
I'd like to extract this to a common file. It's a copy of `cmd/generate/vault.go`